integration/spark/src/main/scala/org/apache/spark/sql/secondaryindex/load/Compactor.scala

```diff
@@ -93,6 +93,9 @@ object Compactor {
             segmentToSegmentTimestampMap, null,
             forceAccessSegment, isCompactionCall = true,
             isLoadToFailedSISegments = false)
+        if (segmentLocks.isEmpty) {
```

Comment:
please add a log here saying that the loading of compacted segment into SI table failed, couldn't acquire segment lock on the specific segment

Reply:
done

Reply:
please add a log, not comment

Reply:
done